### PR TITLE
Add styles for catalog header and categories

### PR DESCRIPTION
**Pull request by cshinn**
screen shot 2017-04-07 at 12 35 35 pm

### CODE REVIEW

**Comment by dtaylor113** [comment]
Since the html/css classes have changed, you'll need to update the unit test:

```
..........................
Firefox 45.0.0 (Fedora 0.0.0) servicesView should display the initial categories and correct number of catalog cards FAILED
	Expected undefined to be 'All'.
	@/home/dtaylor/Dev/repos/openshift/origin-web-catalog/test/test-bundle.ts:9:8007554
.....
```

**Comment by dtaylor113** [comment]
Is that light blue bar, under the light blue underscore, intentional?  IMO it kind of hides the underscore under the currently selected tab.
image

**Comment by dtaylor113** [comment]
At large screen sizes the landing-body-area should probably have a min-height so the light blue background extends to the bottom of the viewport and hides the light gray background.   Not sure if this looks different when viewed from within the console?
image

**Comment by cshinn** [comment]
@dtaylor113:  @rhamilto is currently working on the subcategory and tiles section so I elected to avoid that part of the code. His PR should address the blue bar issue

**Comment by dtaylor113** [comment]
> @dtaylor113: @rhamilto is currently working on the subcategory and tiles section so I elected to avoid that part of the code. His PR should address the blue bar issue

ok, thanks

File: src/styles/services-view.less [hunk 1]
```diff
@@ -8,19 +8,37 @@
...
+    padding-left: (@grid-gutter-width / 2);
+    a {
+      color: @color-pf-white;
+      cursor: pointer;
+
+      &:hover {
+        color: @color-pf-blue-300;
+
+        &:before{
+          content: none;
```
Comment (rhamilto) [inline src/styles/services-view.less]: This makes the :before disappear on :hover of the .active item as well, which feels odd.  Adding a `display: block` on line 40 resolves.

**Comment by cshinn** [comment]
@rhamilto the underline no longer disappears on hover

**Comment by spadgett** [comment]
@cshinn needs rebase

**Comment by serenamarie125** [comment]
@cshinn can you add a screenshot hovering over an item which isn't selected?  Thanks!

**Comment by spadgett** [comment]
@serenamarie125 I have the PR pulled down, so I grabbed a screen

screen shot 2017-04-07 at 12 51 48 pm

**Comment by serenamarie125** [comment]
great thanks @spadgett !  Everything "looks" great thanks @cshinn !

**Pull request merged**